pynamodb/connection/base.py

```diff
@@ -879,6 +893,8 @@ def put_item(self,
         """
         Performs the PutItem operation and returns the result
         """
+        self._check_condition('condition', condition, expected, conditional_operator)
```

Comment:
Why do you need to pass in the string `'condition'`? Is it just for the error message?